lib/components/narrative/line-itin/connected-itinerary-body.js

```diff
@@ -126,6 +131,18 @@ class ConnectedItineraryBody extends Component {
       }
     }
 
+    const LegIconWithA11y = (props) => {
```

Comment:
Move this outside of the class, so that a component class is not recreated on each render.

Reply:
Thanks, good to know for the future too. Added in [fb14b17](https://github.com/opentripplanner/otp-react-redux/pull/825/commits/fb14b17cf53c6d6d939e9f593cca66cfd97917f2)!